Crashlytics/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # v4.0.0-beta.3
 
-- [fixed] Fixed symbol collisions with the legacy Fabric Crashlytics SDK and added a warning not to include both (#4753, #4755)
+- [fixed] Fixed an import declaration when installing using CocoaPods with the `generate_multiple_pod_projects` flag set to true (#4786)
```

Comment:
Please change to:

"Fixed an import declaration for installing Crashlytics using CocoaPods. Previously, the declaration caused a compile error when you installed using CocoaPods with the `generate_multiple_pods_project` flag set to true"

I'm not sure whether importing <FirebaseCore/FIRApp.h> is the same as "installing Crashlytics" so please revise if necessary.

Reply:
Revision makes sense!